test/unit/core/core.js

```diff
@@ -1,4 +1,41 @@
 suite('Core', function(){
+  var myp5, node;
+  
+  setup(function () {
+    node = document.createElement('div');
+    document.body.appendChild(node);
+  });
+  
+  teardown(function () {
+    document.body.removeChild(node);
+  });
+
+  suite('new p5(sketch, null, true)', function () {
+    myp5 = new p5(function(sketch) { }, null, true);
+    myp5.background(128); // sync call
```

Comment:
This looks a little weird... I wanted to put lines 14 and 15 inside the "test" block but it was failing when it shouldn't have been.  I wanted to debug, but for some reason test/test.html doesn't run in the browser.  I spent some time trying to debug but there appears to be a bug inside mocha.js.